elser/operations/weighted-token-queries.json

```diff
@@ -0,0 +1,2511 @@
+{
```

Comment:
I think this is a possible approach but I wonder if we should add a version or replace this one with a live one that generates the weighted terms at query time. That's not possible for the ingestion since it would take too much time on a single machine to index the full dataset but we should be able to infer this cost at query time. We can make it an option of the track as well. Anyway just ideas for follow ups and how we could use this benchmark to test the end to end experience.

Reply:
Generating the tokens at query time requires some orchestration as the ELSER model first needs to be downloaded and deployed then we have to call out to the _infer API. The tokens won't change for the current version of ELSER, hard coding the queries is good for now.

I agree we can be more sophisticated: perhaps pre-generate the tokens for thousands of queries, save those is a JSON file  and generate the weighted term query dynamically from Python. 

The AppEx QA team are testing ELSER end to end and benchmarking how long it takes to ingest their document set. We have good coverage in AppEx and the tooling they use is better suited end to end testing if we wanted to extend that coverage.